spring-cloud-app-broker-core/src/test/java/org/springframework/cloud/appbroker/workflow/instance/AppDeploymentInstanceWorkflowTest.java

```diff
@@ -18,10 +18,7 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.springframework.cloud.appbroker.deployer.BackingApplication;
-import org.springframework.cloud.appbroker.deployer.BackingApplications;
-import org.springframework.cloud.appbroker.deployer.BrokeredService;
-import org.springframework.cloud.appbroker.deployer.BrokeredServices;
+import org.springframework.cloud.appbroker.deployer.*;
```

Comment:
Applying the formatter in https://github.com/spring-cloud/spring-cloud-app-broker/tree/master/src should avoid wildcard imports.

Reply:
thanks, fixed in 45e60b5